lib/devise/models/confirmable.rb

```diff
@@ -125,6 +129,16 @@ def send_reconfirmation_instructions
         end
       end
 
+      def send_new_email_confirmation_instructions
```

Comment:
Do we call this method anywhere in confirmation workflow? We should be triggering the new notification inside `send_confirmation_instructions` (instead of the existing `confirmation_instructions`).